frontend/src/metabase-lib/types.ts

```diff
@@ -224,67 +242,62 @@ export type StringFilterOptions = {
   "case-sensitive"?: boolean;
 };
 
-export type NumberFilterOperatorName = typeof NUMBER_FILTER_OPERATORS[number];
-
 export type NumberFilterParts = {
-  operator: NumberFilterOperatorName;
+  operator: FilterOperator;
   column: ColumnMetadata;
   values: number[];
 };
 
-export type BooleanFilterOperatorName = typeof BOOLEAN_FILTER_OPERATORS[number];
+export type CoordinateFilterParts = {
+  operator: FilterOperator;
+  column: ColumnMetadata;
+  longitudeColumn?: ColumnMetadata;
```

Comment:
btw, the logic is that if there is just one longitude column in the data source, it's omitted (the column is duplicated in mbql). But if there are multiple ones you can select which one should be used

Reply:
Should we maybe add a comment about it somewhere? The way it works isn't really straightforward